lib/output.js

```diff
@@ -91,8 +90,6 @@ function getReporter(format) {
 
 const convertToPercentum = num => num * 100;
 
-const getThreshold = threshold => (typeof threshold === 'number' && threshold && threshold < 100) ? threshold : THRESHOLD;
```

Comment:
Since we're pulling this function out, does it make sense to pull out `getReporter()` as well? It is doing the same sort of thing.

Reply:
Of course, no problem. I'll land this.

Reply:
Hi @exterkamp this request has been done.